plugins/query-insights/src/main/java/org/opensearch/plugin/insights/rules/model/Attribute.java

```diff
@@ -43,7 +43,15 @@ public enum Attribute {
     /**
      * The node id for this request
      */
-    NODE_ID;
+    NODE_ID,
+    /**
+     * User associated with this request
+     */
+    USER_NAME,
+    /**
+     * Custom tenant tags
+     */
+    CUSTOMIZED_TAG;
```

Comment:
We have not populated this anywhere in the `QueryInsightsListener`? What is the plan for this?

Reply:
Will move all them into the one single tag.